keyboards/kyria/keymaps/gotham/oled.c

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#include "keycodes.h"
+
+#ifdef ENCODER_ENABLE
+#    include "encoder.c"
```

Comment:
In general you should not `#include` .c files. Create a header file that declares the stuff you need in this file, and include that instead, so that it isn't potentially compiled twice. You should probably also rename them so as to not conflict with QMK's encoder.[ch].

Reply:
Done!